azure-pipelines-compliance.yml

```diff
@@ -207,13 +220,6 @@ jobs:
     condition: succeededOrFailed()
     continueOnError: True
 
-  # Analyze managed C#/VB code (.NET framework and .NET Core) for security vulnerabilities
```

Comment:
Why's this task removed?

Reply:
Because CodeQL now handles it, the instructions at https://devdiv.visualstudio.com/DevDiv/_wiki/wikis/DevDiv.wiki/1761/Static-Analysis say we should remove all other tools such as Roslyn Analyzer, ESLint, etc...